.cirrus.yml

```diff
@@ -152,13 +152,31 @@ build_and_release_dotnet_task:
     SOLUTION_DIR: ${PROJECT_DIR}\\SonarLint.Secrets.DotNet.sln
     PIPELINE_SCRIPTS_DIR: ${PROJECT_DIR}\\pipeline_scripts
     PROJECT_NAME: "sonar-secrets-dotnet"
+    SNK_PATH: ${CIRRUS_WORKING_DIR}\\SonarSourceSecret.snk
+    ENCODED_SONARSOURCE_SNK: VAULT[development/team/languages/kv/data/strong_named_key data.SonarSourceSecret_snk]
+    ENCODED_SONARSOURCE_SNK_PATH: ${CIRRUS_WORKING_DIR}\\Encoded_SonarSourceSecret.snk
+    SIGN_PASSPHRASE: VAULT[development/kv/data/sign/dotnet data.passphrase]
+    PFX_PATH: ${CIRRUS_WORKING_DIR}\\SonarSource-2021-2023.pfx
+    ENCODED_PFX_CERT: VAULT[development/kv/data/sign/dotnet data.cert_pfx]
+    ENCODED_PFX_CERT_PATH: ${CIRRUS_WORKING_DIR}\\Encoded_SonarSource-2021-2023.pfx
+  sign_cert_pfx_file:
+    path: ${ENCODED_PFX_CERT_PATH}
+    variable_name: ENCODED_PFX_CERT
+  sonarSourceSecret_snk_file:
+    path: ${ENCODED_SONARSOURCE_SNK_PATH} # this is the base64 encoded file
+    variable_name: ENCODED_SONARSOURCE_SNK
+  base64_decode_files_script: |
+    base64 --decode ${ENCODED_SONARSOURCE_SNK_PATH} > ${SNK_PATH}
```

Comment:
Originally I wanted to have only 1 file without encoding in its name and overwriting itself during the decoding, however, I could not do it. I always got an empty file, so I opted for this 2-file solution.